client/pages/searchResults.jsx

```diff
@@ -11,7 +11,12 @@ export default class SearchResults extends React.Component {
   }
 
   componentDidMount() {
-    fetch(`/api/restaurants?${this.props.params.toString()}`)
+    fetch(`/api/restaurants?${this.props.params.toString()}`, {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json'
+      }
```

Comment:
You can leave this configuration out. The default method is GET and there is no body being sent in this request, so the Content-Type isn't needed.